src/styles/index.scss

```diff
@@ -1,3 +1,244 @@
-body {
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+$white-accent: #fff;
+$card-title-color: #34568b;
+
+.catalog {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: $main-accent;
   margin: 0;
+
+  &__header {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0 50px;
+    height: 60px;
+    box-sizing: border-box;
+  }
+
+  &__nav-logo {
+    width: 40px;
+    height: 40px;
+    display: flex;
+    align-items: center;
+  }
+
+  &__nav {
+    display: flex;
+    align-items: center;
+  }
+
+  &__nav-links {
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  &__nav-list {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    line-height: 60px;
+  }
+
+  &__nav-item {
+    display: inline-block;
+    margin-left: 17px;
+    text-align: center;
+    height: 60px;
+    line-height: 60px;
+  }
+
+  &__nav-item:first-child {
+    margin-left: 0;
+  }
+
+  &__nav-link {
+    text-transform: uppercase;
+    text-decoration: none;
+    cursor: pointer;
+    display: flex;
+    color: black;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), colors should be consistent throughout the project. Here, `color: black;` is used directly instead of using a variable like other colors. Consider defining a color variable for black to maintain consistency.